client/src/stories/ForecastChart.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { ForecastChart } from "../components/ForecastChart.js";
+
+export default {
+  title: "Carousel",
+  component: ForecastChart,
+};
+
+const Template = () => (
+  <Router>
```

Comment:
You could think about adding a global decorator to add this Router to every story.
E.g. https://github.com/lmachens/logs/blob/master/client/.storybook/preview.js